include/nbl/builtin/hlsl/scene/animtation.hlsl

```diff
@@ -0,0 +1,42 @@
+
+// Copyright (C) 2018-2022 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#ifndef _NBL_HLSL_SCENE_ANIMATION_INCLUDED_
+#define _NBL_HLSL_SCENE_ANIMATION_INCLUDED_
+
+
+#include <nbl/builtin/hlsl/math/keyframe.hlsl>
+
+namespace nbl
+{
+	namespace hlsl
+	{
+		namespace scene
+		{
+			struct Animation_t

```

Comment:
you want to remove the excessive indentation, let `struct` begin at a line with no indentation